src/aggregation/CountAggregator.js

```diff
@@ -5,35 +5,30 @@ var util = require('../util');
 var ad = require('../ad');
 var dists = require('../dists');
 
-var CountAggregator = function() {
-  this.hist = {};
+var CountAggregator = function(onlyMAP) {
+  this.onlyMAP = onlyMAP;
+  this.max = {value: undefined, score: -Infinity};
+  this.samples = [];
 };
 
-CountAggregator.prototype.add = function(value) {
-  var k = util.serialize(value);
-  if (this.hist[k] === undefined) {
-    this.hist[k] = { count: 0, val: value };
+CountAggregator.prototype.add = function(value, score) {
+  score = (score === undefined) ? 0 : score;
```

Comment:
I think I'd go with your suggestion to omit the score property when we don't have it. (Can happen in a follow-up PR if it's not a quick change.)